pkg/volume/plugins.go

```diff
@@ -266,3 +293,64 @@ func (pm *VolumePluginMgr) FindRecyclablePluginBySpec(spec *Spec) (RecyclableVol
 	}
 	return nil, fmt.Errorf("no recyclable volume plugin matched")
 }
+
+// createDefaultScrubberPodTemplate creates a template for a scrubber pod.  Most attributes are correct for scrubbers,
+// but other plugins are required to change the VolumeSource.  Plugins should also consider changing the pod's
+// ActiveDeadlineSeconds timeout and GenerateName. See the NFS plugin and its overrides.
+func createDefaultScrubberPodTemplate() *api.Pod {
+	timeout := int64(60)
+	pod := &api.Pod{
+		ObjectMeta: api.ObjectMeta{
+			GenerateName: "pv-scrubber-hostpath-",
+			Namespace:    api.NamespaceDefault,
+		},
+		Spec: api.PodSpec{
+			ActiveDeadlineSeconds: &timeout,
+			RestartPolicy:         api.RestartPolicyNever,
+			Volumes: []api.Volume{
+				{
+					Name: "vol",
+					// IMPORTANT!  All plugins using this template must override the VolumeSource
+					// and make it applicable to the PersistentVolume being recycled.
+					// See HostPath and NFS implementations for examples.
+					VolumeSource: api.VolumeSource{
+						HostPath: &api.HostPathVolumeSource{"/thePathToScrub"},
```

Comment:
I can remove that from here (but leave the comment and instructions) and likewise make sure to strip the CLI pod (if provided as an override) of the same.  The recycler can then validate that a real VolumeSource was correctly added by the plugin.